gemfile.d/test.rb

```diff
@@ -0,0 +1,7 @@
+group :test do
+  gem "minitest-tags"
+  gem "vcr", "< 4.0.0"
+  gem "rubocop-checkstyle_formatter"
+  gem "simplecov"
+  gem "simplecov-rcov"
```

Comment:
I was considering a patch to make simplecov loading optional. If we don't publish the coverage, is there any point to having it? Or really, does anyone look at it?

Reply:
My vote is to remove simplecov. @ianballou @parthaa please weigh in

Reply:
Let's move that over to https://github.com/Katello/katello/pull/10837.